kumascript/macros/LiveSampleURL.ejs

```diff
@@ -4,25 +4,27 @@
 //
 // Parameters:
 //
-// $0   The ID of the sample's section or containing block.
-// $1   The URL from which to pull the sample; if not provided,
+// $0 - The ID of the sample's section or containing block.
+// $1 - (Deprecated)  The URL from which to pull the sample; if not provided,
 //      the current page is assumed.
 
-var pagePath = env.url;
+let pagePath = env.url;
+let baseUrl = env.live_samples.base_url;
+let samplePath = `${pagePath}/runner.html`.replace(/\/\/+/g, "/");
 
 if ($1 && $1.length > 0) {
```

Comment:
Since you're cleaning up the macro anyways, can you assign $0 and $1 to variables `id` and `url` or similar to avoid referencing $0 and $1 below?